rules/cross-platform/lateral_movement_malicious_remote_file_creation.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2023/10/12"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
```

Comment:
This should be 8.9.0?

Reply:
These rules are not bound to the integration are they?

Reply:
Got it. So I will update the min-stack to avoid rule duplication for <= 8.8 stack versions.